Sources/MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -418,7 +425,7 @@ open class DirectionsOptions: NSObject, NSSecureCoding, NSCopying {
 
      By default, no attribute options are specified. It is recommended that `routeShapeResolution` be set to `.full`.
      */
-    @objc open var attributeOptions: AttributeOptions = []
+    open var attributeOptions: MBAttributeOptions = []
```

Comment:
Can the type remain as `AttributeOptions`, or does it have to be downgraded to `MBAttributeOptions`?